<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoYamsClass</class>
 <widget class="QMainWindow" name="AutoYamsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoYams</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="pagination">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_menu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="HLayout_1" stretch="1,3,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="VWidget_1_1" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="VLayout_1_1">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QListWidget" name="menu_playerList"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="HLayout_1_1_1">
               <item>
                <widget class="QLineEdit" name="menu_playerName"/>
               </item>
               <item>
                <widget class="QPushButton" name="menu_playerAdd">
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="menu_startBtn">
               <property name="text">
                <string>Start Game</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_game">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="VLayout_2" stretch="0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QTableWidget" name="game_scoreTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>true</bool>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <property name="rowCount">
             <number>20</number>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>Joueur</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <color>
               <red>51</red>
               <green>176</green>
               <blue>204</blue>
              </color>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="255">
                <red>223</red>
                <green>185</green>
                <blue>33</blue>
               </color>
              </brush>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDe1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDe2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDe3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDe4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDe5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDe6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>TotalDes</string>
             </property>
             <property name="background">
              <color>
               <red>234</red>
               <green>234</green>
               <blue>234</blue>
              </color>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Bonus</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Total1</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <color>
               <red>182</red>
               <green>182</green>
               <blue>182</blue>
              </color>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Separation</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Brelan</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Carre</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Full</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>PetiteSuite</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>GrandeSuite</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Yams</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Chance</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Total2</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <color>
               <red>182</red>
               <green>182</green>
               <blue>182</blue>
              </color>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Total</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <color>
               <red>132</red>
               <green>132</green>
               <blue>132</blue>
              </color>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Display</string>
             </property>
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>JOUEURS</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>25</red>
                <green>186</green>
                <blue>189</blue>
               </color>
              </brush>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="255">
                <red>231</red>
                <green>207</green>
                <blue>21</blue>
               </color>
              </brush>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>Total de 1</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>Total de 2</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>Total de 3</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>Total de 4</string>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string>Total de 5</string>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string>Total de 6</string>
             </property>
            </item>
            <item row="7" column="0">
             <property name="text">
              <string>Total</string>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="background">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>222</red>
                <green>222</green>
                <blue>222</blue>
               </color>
              </brush>
             </property>
            </item>
            <item row="8" column="0">
             <property name="text">
              <string>Bonus (&gt;63)</string>
             </property>
            </item>
            <item row="9" column="0">
             <property name="text">
              <string>Total I</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>185</red>
                <green>185</green>
                <blue>185</blue>
               </color>
              </brush>
             </property>
            </item>
            <item row="10" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="11" column="0">
             <property name="text">
              <string>Brelan (total des 3 dés)</string>
             </property>
            </item>
            <item row="12" column="0">
             <property name="text">
              <string>Carré (total des 4 dés)</string>
             </property>
            </item>
            <item row="13" column="0">
             <property name="text">
              <string>Full (25 points)</string>
             </property>
            </item>
            <item row="14" column="0">
             <property name="text">
              <string>Petite Suite (30 points)</string>
             </property>
            </item>
            <item row="15" column="0">
             <property name="text">
              <string>Grande Suite (40 points)</string>
             </property>
            </item>
            <item row="16" column="0">
             <property name="text">
              <string>Yams (50 points)</string>
             </property>
            </item>
            <item row="17" column="0">
             <property name="text">
              <string>Chance (total 5 dés)</string>
             </property>
            </item>
            <item row="18" column="0">
             <property name="text">
              <string>Total II</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>185</red>
                <green>185</green>
                <blue>185</blue>
               </color>
              </brush>
             </property>
            </item>
            <item row="19" column="0">
             <property name="text">
              <string>TOTAL</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="background">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>145</red>
                <green>145</green>
                <blue>145</blue>
               </color>
              </brush>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="game_diceDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>580</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Dices</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="DiceContainer_HLayout_2_1" stretch="1,1,1,1,1">
               <item>
                <widget class="QLabel" name="game_dice_1">
                 <property name="text">
                  <string>Dice</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="game_dice_2">
                 <property name="text">
                  <string>Dice</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="game_dice_3">
                 <property name="text">
                  <string>Dice</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="game_dice_4">
                 <property name="text">
                  <string>Dice</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="game_dice_5">
                 <property name="text">
                  <string>Dice</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="Frame_2_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>580</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="HLayout_2_1" stretch="1,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <widget class="QListWidget" name="game_bestMoves">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="Frame_2_1_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QGridLayout" name="GLayout_2_1_1">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMaximumSize</enum>
                </property>
                <item row="2" column="0">
                 <widget class="QPushButton" name="game_nextThrow">
                  <property name="text">
                   <string>Rethrow</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="game_currentThrow">
                  <property name="text">
                   <string>none</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QComboBox" name="game_moveCB">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="game_nextPlayer">
                  <property name="text">
                   <string>Accept</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="game_currentPlayer">
                  <property name="text">
                   <string>none</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="game_currentTurn">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>none</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuInput">
     <property name="title">
      <string>Input</string>
     </property>
     <addaction name="actionAuto"/>
     <addaction name="actionSemi"/>
     <addaction name="actionManual"/>
    </widget>
    <addaction name="actionVideo"/>
    <addaction name="menuInput"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSource"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionSemi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Semi-Automatic</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="text">
    <string>Source</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutoYams.qrc"/>
 </resources>
 <connections/>
</ui>
